composer.json

```diff
@@ -24,7 +24,8 @@
   "autoload": {
     "psr-4": {
       "Terminus\\": ["php/", "php/Terminus/"],
-      "Pantheon\\Terminus\\": "src/"
+      "Pantheon\\Terminus\\": "src/",
+      "Pantheon\\Terminus\\Tests\\": "tests/"
```

Comment:
I usually prefer to make this `tests/src` so that your test classes are not mixed in with your other testing files.